packages/big-design/src/components/Worksheet/hooks/useUpdateItems/spec.tsx

```diff
@@ -1,9 +1,27 @@
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react';
+import { Component, PropsWithChildren } from 'react';
 
 import { useUpdateItems } from './index';
 
+beforeEach(() => {
+  // We need to suppress console errors, otherwise we'll blow up the console.
+  // eslint-disable-next-line no-console
+  console.error = jest.fn();
+});
+
 test('throws if is not wrapped in UpdateItemsProvider', () => {
-  const { result: hook } = renderHook(() => useUpdateItems());
+  let error;
+
+  renderHook(() => useUpdateItems(), {
+    wrapper: class Wrapper extends Component<PropsWithChildren<unknown>> {
+      override componentDidCatch(err: unknown) {
+        error = err;
+      }
+      override render() {
+        return this.props.children;
+      }
+    },
+  });
 
```

Comment:
When they ported over `renderHook` to v13 of `@testing-library/react` they shipped it with a leaner API. Now we don't get errors with this, but according to https://github.com/testing-library/react-testing-library/pull/991#issuecomment-1207138334 this is the best way to achieve the same thing.

Reply:
😵‍💫